### PR TITLE
Enable group settings for everyone using _wcpay_feature_grouped_settings

### PR DESCRIPTION
**Pull request by harriswong**
Fixes https://github.com/Automattic/woocommerce-payments/issues/1990

#### Changes proposed in this Pull Request
Enable the new WooCommerce Payments settings page by defaulting our existing feature flag `_wcpay_feature_grouped_settings` to `1`.

Before
image

After
image

#### Testing instructions
1. Remove this option from your database if you have one `_wcpay_feature_grouped_settings`. `DELETE FROM `wp_options` WHERE option_name = '_wcpay_feature_grouped_settings'`
2. Go to WooCommerce > Settings > Payments (`wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments`), you should see the new layout above.

-------------------

- [x] Added changelog entry (or does not apply)
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

File: includes/class-wc-payments-features.php [hunk 1]
```diff
@@ -19,7 +19,7 @@ class WC_Payments_Features {
 	 * @return bool
 	 */
 	public static function is_grouped_settings_enabled() {
-		return '1' === get_option( '_wcpay_feature_grouped_settings', '0' );
+		return '1' === get_option( '_wcpay_feature_grouped_settings', '1' );
```
Comment (harriswong) [inline includes/class-wc-payments-features.php:22]: Per discussion with @waclawjacek and @frosso , we are keeping this feature flag but defaulting it to `1`. This technique allows us to display the new layout without adding a new entry into the DB option table. Worst case, we can roll out patches to disable the feature by manipulating `_wcpay_feature_grouped_settings`.

**Review by frosso — APPROVED** [review]
Looks straightforward - let's do this 👍

**Comment by harriswong** [comment]
Fixed merge conflicts, rebased.

**Comment by harriswong** [comment]
pipeline ran ok. merging!

**Pull request merged**